examples/hubtype_demo/README.md

```diff
@@ -0,0 +1,49 @@
+## Set up
+
+npm i -g contentful-cli
+npm i package.json
+
+### Import space to Contentful:
+
+Create an account into Contentful
```

Comment:
I would add.
`See https://docs.botonic.io/plugins/plugin-contentful for more details`